httptransport/matcher_v1.go

```diff
@@ -125,9 +126,10 @@ func (h *MatcherV1) vulnerabilityReport(w http.ResponseWriter, r *http.Request)
 	if err != nil {
 		apiError(ctx, w, http.StatusInternalServerError, "experienced a server side error: %v", err)
 	}
-	// now check bool only after confirming no err
-	if !ok {
+	// now check present and finished only after confirming no err
+	if !ok || indexReport.State != indexerController.IndexFinished.String() {
```

Comment:
nit but it might be nicer to compare the state types as opposed to the strings:
```go
var state controller.State
state.FromString(indexReport.State)
if !ok || state != controller.IndexFinished {
...
```